executor/executor_test.go

```diff
@@ -5565,7 +5565,7 @@ func TestAdmin(t *testing.T) {
 	tk = testkit.NewTestKit(t, store)
 	tk.MustExec("begin")
 	sess := tk.Session()
-	ddlInfo, err := ddl.GetDDLInfo(sess)
+	ddlInfo, err := ddl.GetDDLInfoWithNewTxn(sess)
```

Comment:
Why update this function？ line5566 will begin a new txn.

Reply:
let me revert it back (I changed `GetDDLInfo` to `getDDLInfo` to make sure I did not miss any cases where new transactions would be needed, but that is no longer needed).

Reply:
Reverted.